# Awesome LatchBio [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)

> Bioinformatics is an interdisciplinary field that develops methods and software tools for understanding biological data. â€” [Wikipedia](https://en.wikipedia.org/wiki/Bioinformatics)

A curated list of awesome Bioinformatics pipelines and software available on [LatchBio](https://console.latch.bio/explore). Command line or programmatic tools are now with Graphical User Interface and can be run without the hassle of software installation! Please feel free to [contribute](CONTRIBUTING.md)!

<!-- START doctoc generated TOC please keep comment here to allow auto update -->
<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
## Table of Contents

- [Genome Assembly](#genome-assembly)
  - [De novo](#de-novo)
- [Multiple sequence alignments (MSAs) and phylogenetics](#multiple-sequence-alignments-msas-and-phylogenetics)
  - [Trim multiple sequence alignment](#trim-multiple-sequence-alignment)
  - [Align multiple sequences](#align-multiple-sequences)
  - [Alignment summary statistics](#alignment-summary-statistics)
  - [Phylogenetic inference](#phylogenetic-inference)
- [Analyses of codons](#analyses-of-codons)
  - [Codon optimization](#codon-optimization)

<!-- END doctoc generated TOC please keep comment here to allow auto update -->

---

### Genome Assembly

#### De novo

- **[shasta](https://github.com/chanzuckerberg/shasta)** - Rapidly produce accurate assembled sequence using DNA reads generated by Oxford Nanopore flow cells as input. [ [paper-2020](https://pubmed.ncbi.nlm.nih.gov/32686750) | [documentation](https://chanzuckerberg.github.io/shasta) ]

- **[genome_assembly_metrics](https://jlsteenwyk.com/BioKIT/)** - calculate diverse summary statistics for a genome assembly using [BioKIT](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183) [ wf.genome_assembly_metrics_biokit | [paper-2022](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183) | [documentation](https://jlsteenwyk.com/BioKIT/) ]

### Multiple sequence alignments (MSAs) and phylogenetics

#### Trim multiple sequence alignment
- **[ClipKIT](https://github.com/JLSteenwyk/ClipKIT)** - multiple sequence alignment trimming for accurate phylogenomic inference. [ wf.clipkit | [paper-2020](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3001007) | [documentation](https://jlsteenwyk.com/ClipKIT/) ]

#### Align multiple sequences
- **[MAFFT](https://mafft.cbrc.jp/alignment/software/)** - multiple sequence alignment program. [ wf.align_sequences_mafft |  [paper-2013](https://academic.oup.com/mbe/article/30/4/772/1073398) | [documentation](https://mafft.cbrc.jp/alignment/software/) ]

#### Alignment summary statistics
- **[alignment_summary_statistics](https://jlsteenwyk.com/BioKIT/)** - calculate diverse summary statistics for a multiple sequence alignment using [BioKIT](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183) [ wf.alignment_summary_statistics | [paper-2022](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183) | [documentation](https://jlsteenwyk.com/BioKIT/) ]

#### Phylogenetic inference
- **[IQ-TREE2](http://www.iqtree.org/doc/)** - efficient software for phylogenomic inference. [ wf.infer_phylogeny | [paper-2020](https://doi.org/10.1093/molbev/msaa015) | [documentation](http://www.iqtree.org/doc/) ]

### Analyses of codons

#### Codon optimization
- **[gRSCU](https://jlsteenwyk.com/BioKIT/)** - estimate codon optimization using [BioKIT](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183) [ wf.codon_optimization_biokit | [paper-2022](https://academic.oup.com/genetics/advance-article-abstract/doi/10.1093/genetics/iyac079/6583183) | [documentation](https://jlsteenwyk.com/BioKIT/) ]